examples/ensemble/plot_stack_predictors.py

```diff
@@ -196,7 +197,9 @@ def load_ames_housing():
     ("Gradient Boosting", gbdt_pipeline),
 ]
 
-stacking_regressor = StackingRegressor(estimators=estimators, final_estimator=RidgeCV())
+stacking_regressor = StackingRegressor(
+    estimators=estimators, cv=2, final_estimator=RidgeCV()
```

Comment:
I think this is okay, as long as we leave a code comment above that states that `cv=2` is used for reducing the runtime of the example.

Reply:
I added a comment, hopefully this is about in line with what you thought about.